packages/core/src/dev/activation.ts

```diff
@@ -61,57 +61,59 @@ let targetAuth: Auth
  * on selection. There is no support for name-spacing. Just add the relevant
  * feature/module as a description so it can be moved around easier.
  */
-const menuOptions: Record<DevFunction, MenuOption> = {
```

Comment:
This was defined at extension load and called `getSessionId()` before globals was defined. So we just make this a function so it is created once needed